generators/wp/index.js

```diff
@@ -41,6 +40,7 @@ var PHPCS_FILE                      = '/tmp/phpcs.json';
 var PHPCS_STANDARD                  = 'PmcWpVip';
 var PHPUNIT_EXCLUDE_GROUPS          = 'pmc-phpunit-ignore-failed';
 var PHP_VERSION                     = '7.3';
+var PMC_PHPUNIT_BOOTSTRAP           = "/var/www/html/wp-content/plugins/pmc-plugins/pmc-unit-test/bootstrap.php"
```

Comment:
@hauvong Can we use the WEB_ROOT var to define the first part of this as a default?